python/foglamp/services/core/api/backup_restore.py

```diff
@@ -118,8 +118,6 @@ def _get_status(status_code):
     :Example: curl -X GET http://localhost:8081/foglamp/backup/1
     """
     backup_id = request.match_info.get('backup_id', None)
-    if not backup_id:
-        raise web.HTTPBadRequest(reason='Backup id is required')
```

Comment:
This code is never called, before call comes here, its handled by routes `404: Not Found`